NetworkingTests/Tests/SessionManagerSpec.swift

```diff
@@ -311,13 +306,6 @@ internal class SessionManagerSpec: QuickSpec {
                     expect(sessionManager.currentUser!.sessionToken).to(equal(updatedSessionToken))
                 }
                 
-                it("does not update the session token") {
```

Comment:
The same I think.

Reply:
@jcaracciolo modified that method in another PR to save the session token, so this test fails. As far as he told me he talked this with you and you were okay with removing both tests, but he forgot to remove them in that PR